powersimdata/utility/tests/test_helpers.py

```diff
@@ -68,6 +68,15 @@ def test_mem_cache_get_returns_copy():
     assert id(cache.get(key)) != id(obj)
 
 
+def test_mem_cache_put_version_never_changes():
+    cache = MemoryCache()
+    key = cache_key("foo", 4)
+    obj = {"key1": 42}
+    cache.put(key, obj)
+    obj["key2"] = 8675309
+    assert "key2" not in cache.get(key)
```

Comment:
Can we simplify this test to make it easier to read such as:
```
def test_mem_cache_put_version_never_changes():
    cache = MemoryCache()
    obj = {"key1": "val1"}
    cache.put("foo", obj)
    obj["key2"] = "val2"
    assert "key2" not in cache.get("foo")
```

Reply:
Sure. I was basically copying and pasting from the test above, but I agree that yours is clearer about what we're trying to do. We could also `assert "key1" in cache.get(key)`.

Reply:
Maybe also `assert obj is not cache.get(key)`?

Reply:
I think `test_mem_cache_get_returns_copy` already catches the first one.

Reply:
Done.